napari/_app_model/_app.py

```diff
@@ -32,7 +30,7 @@ def __init__(self, app_name=APP_NAME) -> None:
 
     @classmethod
     def get_app_model(cls, app_name: str = APP_NAME) -> NapariApplication:
-        return Application.get_app(app_name) or cls()  # type: ignore[return-value]
+        return Application.get_app_model(app_name) or cls()  # type: ignore[return-value]
```

Comment:
This change is incorrect and might be the main source of failing CI?

Reply:
It is, it's my fault. there was a different issue before and i misread things on my phone. 😞

Reply:
Clarified with a docstring.